Makefile

```diff
@@ -30,7 +30,12 @@ superset:
 	pip install -e .
 
 	# Create an admin user in your metadata database
-	superset fab create-admin
+	superset fab create-admin \
+                    --username admin \
+                    --firstname Admin \
+                    --lastname Strator \
+                    --email admin@superset.io \
+                    --password admin
```

Comment:
please change the password to "general" 
or change all the test fixtures that add an admin user, the password to admin